shablona/data.py

```diff
@@ -0,0 +1,144 @@
+"""shablona.data: download and read data."""
+import os
+import sys
+import contextlib
+import os.path as op
+from hashlib import md5
+from shutil import copyfileobj
+
+if sys.version_info[0] < 3:
+    from urllib2 import urlopen, urljoin
+else:
+    from urllib.request import urlopen, urljoin
+
+# Set a user-writeable file-system location to put files:
+SHABLONA_HOME = op.join(os.path.expanduser('~'), '.shablona')
```

Comment:
Also, I tend to avoid putting data in a hidden directory--I use those for settings only, in general. Just a note about my preference, no request for change :)